src/models/TestStationDAO.ts

```diff
@@ -74,7 +74,22 @@ export class TestStationDAO {
   }
 
   /**
-   * Write data about multiple Test Stations to the DB.
+   * Insert or update a Test Station in the DB.
+   * @param testStationItem: ITestStation
+   * @returns DynamoDB PutItemOutput, wrapped in promises
+   */
+  public putItem(
+    testStationItem: ITestStation
+  ): Promise<PromiseResult<DocumentClient.PutItemOutput, AWS.AWSError>> {
+    const params = {
+      TableName: this.tableName,
```

Comment:
Don't we need to have key in this?

Reply:
Doesn't seem to need it. If you omit testStationId from the message dynamodb rejects the put. When the validation work is done, a missing testStationId value will be picked up before it gets to this stage.